parity/cli/mod.rs

```diff
@@ -348,6 +348,10 @@ usage! {
 			"--unlock=[ACCOUNTS]",
 			"Unlock ACCOUNTS for the duration of the execution. ACCOUNTS is a comma-delimited list of addresses.",
 
+			ARG arg_enable_signing_queue: (bool) = false, or |c: &Config| c.account.as_ref()?.enable_signing_queue,
+			"--enable-signing-queue=[BOOLEAN]",
```

Comment:
I'm in favor renaming it to `--enable-external-signing`, I think signing queue might be hard to grasp as a user?!

Reply:
Personally, don't have a preference. cc @dvdplm @ordian thoughts?

Reply:
I think @niklasad1 is right and `--enable-external-signing` is a better name; however I'm not sure it's "enough betterness" to warrant the hassle of supporting both with deprecation notices and all. So yeah, I'd probably keep it.
It **does** need a better documentation string though (and likely some wiki edits as well).

Reply:
where's the wiki?

Reply:
https://github.com/paritytech/wiki/

Reply:
I don't have a strong opinion, but `--enable-external-signing` sounds more user-friendly indeed

Reply:
> where's the wiki?

https://github.com/paritytech/wiki (published every x minutes as https://wiki.parity.io/Parity-Ethereum).